README.rst

```diff
@@ -113,13 +113,14 @@ The main idea behind this is that they are not shared by cells - you need to hav
 
     a) First column contains sprint number (you can create multiple role tasks for one week by dividing sprint into parts, e.g. `Sprint 100a, Sprint 100b` - each in a separate row).
     b) Next columns' headers contain role names prefixed by the full cell name (e.g. `Cell_1 FF`) and their fields contain assignees for the tickets.
+    c) The "Date" column is omitted.
 
-    The metadata (name, of these ticket is defined in `JIRA_CELL_ROLES`. Please see its docstring for the detailed explanation of its format.
+    The metadata (name, duration, story points), of these tickets is defined in `JIRA_CELL_ROLES`. Please see its docstring for the detailed explanation of its format.
```

Comment:
@Agrendalath Thanks for updating this. There's still an extra comma after `)` that's not needed. Let's remove it :)